csharp/beetlex/web.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
what is this file for ?

Reply:
so you can open it in Jetbrains Rider or Visual Studio

Reply:
ah ok, this is a file like `.vscode`, an IDE file

could it be removed @beetlex-io (in order no to upload on container) ?